### ISSUE TITLE
Feedback

### ISSUE THREAD

**Issue by bwreid** [abglassford/week-1-project#1]
- I would leave your `index.html` file at the root level. Specifically, when we're deploying static sites we'll want it there so that our sites will pick up the page correctly!
- The HTML does not validate correctly.
- [index.html, LN 6-7](https://github.com/abglassford/week-1-project/blob/master/index/index.html#L6-L7): Typically you want your CSS after any external stylesheets.
- [index.html, LN 37-41](https://github.com/abglassford/week-1-project/blob/master/index/index.html#L37-L41): Looks like your indentation is getting a little wonky.
- [index.html, LN 41](https://github.com/abglassford/week-1-project/blob/master/index/index.html#L41): Do you mean for your `nav` to be wrapped by the `a` element? Typically it'd be the other way around. Same thing in [your header nav](https://github.com/abglassford/week-1-project/blob/master/index/index.html#L41)
- [main.css, LN 65-76](https://github.com/abglassford/week-1-project/blob/master/index/index.html#L6-L7): You're duplicating code here for the asides. Since they're all the same, you can just apply the same class to all of them. Or, select the element.
- You're not actually using `clear: both;` after the middle section with the image. Implicitly, it looks like it's working but if something were to change (e.g. the image link is broken), the rest of the content below would float up and mess up the view. To see, try changing the image width to 20%.
- I would try adding some comments to split up your CSS. Think about how to sectionalize it so it's easier to read.

Great job on splitting up your images and styles into separate folders and using semantic tags! Make the above changes and re-push back up to github.

**Comment by abglassford** [comment]
I tried to do all the changes you recommended except for the comments (I pushed the newer version up before I added them. I'll add them soon.), and what you said about the `clear: both;` in the middle section. I must have worked on it a little bit after I submitted the version you looked at, because I couldn't find the a clear in my CSS. However, when I resize my image, it still screws up the page. Is there something I can do to fix that?
